rest/route/service.go

```diff
@@ -33,6 +33,7 @@ func AttachHandler(app *gimlet.APIApp, opts HandlerOpts) {
 	sc.SetURL(opts.URL)
 
 	// Middleware
+	requireUserToggleable := NewRequireAuthHandler()
```

Comment:
Can we link a TODO for a ticket to eventually replace these with require user?

Reply:
Added: https://jira.mongodb.org/browse/EVG-17989